### PR TITLE
Documentation/op-guide: Add rules for Prometheus 2.0

### PR DESCRIPTION
**Pull request by brancz**
This adds the existing Prometheus alerting rules in the new Prometheus 2.0 rule syntax.

@xiang90 @gyuho

### CODE REVIEW

File: Documentation/op-guide/etcd3_alert.rules.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  - alert: HighNumberOfLeaderChanges
+    expr: increase(etcd_server_leader_changes_seen_total{job="etcd"}[1h]) > 3
+    labels:
+      severity: warning
+    annotations:
+      description: etcd instance {{ $labels.instance }} has seen {{ $value }} leader
+        changes within the last hour
+      summary: a high number of leader changes within the etcd cluster are happening
+  - alert: HighNumberOfFailedGRPCRequests
+    expr: sum(rate(etcd_grpc_requests_failed_total{job="etcd"}[5m])) BY (grpc_method)
```
Comment (gyuho) [inline Documentation/op-guide/etcd3_alert.rules.yml]: Actually, should we update this to our new go-grpc-prometheus metrics?
https://github.com/coreos/etcd/issues/8802

/cc @xiang90

Comment (brancz) [reply Documentation/op-guide/etcd3_alert.rules.yml]: Yes, I was going to do that in a subsequent PR, happy to do it here as well though.

Comment (gyuho) [reply Documentation/op-guide/etcd3_alert.rules.yml]: > Yes, I was going to do that in a subsequent PR

Ok sounds good.
Can you file another PR?

Thanks!

File: Documentation/op-guide/monitoring.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ Now Prometheus will scrape etcd metrics every 10 seconds.
 
 ### Alerting
 
-There is a [set of default alerts for etcd v3 clusters](./etcd3_alert.rules).
+There is a set of default alerts for etcd v3 clusters for [Prometheus 1.x](./etcd3_alert.rules) as well as [Prometheus 2.x](./etcd3_alert.rules).
```
Comment (xiang90) [inline Documentation/op-guide/monitoring.md]: one of the links is wrong. they point to the same file now. the v2 one should have .yam suffix?

/cc @gyuho @brancz

Comment (brancz) [reply Documentation/op-guide/monitoring.md]: You're absolutely right. I'll fix it with the follow-up PR!

File: Documentation/op-guide/etcd3_alert.rules.yml (hunk 1 above)
Comment (brancz) [reply Documentation/op-guide/etcd3_alert.rules.yml]: Will do.

File: Documentation/op-guide/monitoring.md (hunk 2 above)
Comment (gyuho) [reply Documentation/op-guide/monitoring.md]: Right,

@brancz when you fix alert rules for https://github.com/coreos/etcd/issues/8802, can we also change `[Prometheus 2.x](./etcd3_alert.rules).` to `[Prometheus 2.x](./etcd3_alert.rules.yml).`?

Thanks!

**Pull request merged**